plugins/org.python.pydev.parser/src/org/python/pydev/parser/jython/ast/Python.asdl

```diff
@@ -114,7 +114,7 @@ module Python
 	-- the Artificial context is not actually used in the parser directly. It is used in the application
 	-- to create 'artificial' names, as if it was a regular name (e.g.: create a name for a match inside
 	-- a comment or string)
-	expr_context = Load | Store | Del | AugLoad | AugStore | Param | Artificial | KwOnlyParam |  NamedStore
+	expr_context = Pattern | Load | Store | Del | AugLoad | AugStore | Param | Artificial | KwOnlyParam |  NamedStore
```

Comment:
New values should be added to the end (so that the numbers assigned don't change).